Makefile

```diff
@@ -32,4 +31,16 @@ all-sidecar:
 	cd sidecar && $(MAKE) all
 
 all-sentinel:
-	cd sentinel && $(MAKE) all
\ No newline at end of file
+	cd sentinel && $(MAKE) all
+
+install-demo:
+	cd demo && $(MAKE) deploy
+
+install-safe:
+	cd safe && $(MAKE) deploy
+
+install-sidecar:
+	cd sidecar && echo "no-op"
```

Comment:
there is nothing to do to install the sidecar; for now at least.

we may create a mutating admission hook or something later and add the command here.